scss/settings/_settings.scss

```diff
@@ -74,7 +74,7 @@ $paragraph-margin-bottom: 10px;
 $small-font-size: 80%;
 $small-font-color: $medium-gray;
 $lead-font-size: $global-font-size * 1.25;
-$lead-line-height: 1.6;
+$lead-line-height: 160%;
```

Comment:
Revert from here and change it here => https://github.com/zurb/foundation-emails/blob/develop/scss/components/_typography.scss#L91